practice_app/api/views.py

```diff
@@ -90,7 +90,8 @@ def google_scholar(request):
         number = 3
     else:
         number = int(number)
-
+    print(search)
+    print(number)
```

Comment:
I think you forgot to delete these lines

Reply:
fixed